DOCS-3456: Raspberry pi ai camera IMX500 not officially supported (#3920)
viamrobotics/docs: docs/operate/reference/components/camera/webcam.md

{{% expand "CSI Camera not working on a Raspberry Pi" %}}
If you are using a CSI camera v1.3 or v2.0 on a Raspberry Pi, you need to [enable legacy mode](/operate/reference/prepare/rpi-setup/#enable-communication-protocols).
If you are using a CSI camera v3.0, you need to use the [`viam:camera:csi` module](https://github.com/viamrobotics/csi-camera/) instead.

> IMX500 AI cameras are not offically supported.

Author: ```suggestion
IMX500 AI cameras are not officially supported.
```

> IMX500 AI cameras are not officially supported.

Author: ```suggestion
For the Raspberry Pi AI cameras like the IMX500 AI camera, you can use this [vision service module](https://github.com/HipsterBrown/viam-pi-ai-camera).
For more information about the vision service, see [run inference](https://docs.viam.com/data-ai/ai/run-inference/).
```

{{% expand "CSI Camera not working on a Raspberry Pi" %}}
If you are using a CSI camera v1.3 or v2.0 on a Raspberry Pi, you need to [enable legacy mode](/operate/reference/prepare/rpi-setup/#enable-communication-protocols).
If you are using a CSI camera v3.0, you need to use the [`viam:camera:csi` module](https://github.com/viamrobotics/csi-camera/) instead.

> For the Raspberry Pi AI cameras like the IMX500 AI camera, you can use this [vision service module](https://github.com/HipsterBrown/viam-pi-ai-camera).

Author: ```suggestion
For the Raspberry Pi AI cameras like the IMX500 AI camera, you can use this [vision service module](https://github.com/HipsterBrown/viam-pi-ai-camera), but this is not officially supported by Viam.
```

> For the Raspberry Pi AI cameras like the IMX500 AI camera, you can use this [vision service module](https://github.com/HipsterBrown/viam-pi-ai-camera), but this is not officially supported by Viam.

Author: added not officiially supported part at av team request

Reviewer: ```suggestion
For Raspberry Pi AI cameras like the IMX500 AI camera, you can use a module such as [this `viam-pi-ai-camera` vision service](https://github.com/HipsterBrown/viam-pi-ai-camera).
Note that this module is not officially supported by Viam.
```
[opt] Might want to hint that there could be more options (they could write their own or someone else might publish one tomorrow), provide more descriptive link text, and word in a way that sounds maybe less discouraging? But this also gets more verbose so up to you!
